src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
```

Comment:
Функція `makeCalculator` має повертати калькулятор
Дієсловами зазвичай називаємо функції 
```suggestion
  const calculator = {
```